spec/8-service-apis.md

```diff
@@ -1,15 +1,17 @@
 # 8 Service APIs
 
-This section describes external APIs that must be implemented by the building block. Additional APIs may be implemented by the building block (all APIs must adhere to the standards and protocols defined), but the listed APIs define a minimal set that must be provided by any implementation.
+This section describes external APIs that must be implemented by the building block. Additional APIs may be implemented by the building block (all APIs must adhere to the standards and protocols defined), but the listed APIs define a minimal set that must be provided by any implementation. 
 
-{% hint style="success" %}
-All APIs will be defined using the OpenAPI (Swagger) standard. The API definitions will be hosted outside of this document. This section may provide a brief description of required APIs.&#x20;
+The current API specification can be found here:
 
-This section will primarily contain links to the GitHub repository for OpenAPI definition (yaml) files as well as to a website hosted by GovStack that provides a live API documentation portal.
+* [API and Data model main definition document](https://docs.google.com/spreadsheets/d/1snIszqyTGYk1u25liwQ_1jONTsQeH7D8aqv1Td74xt4/edit?usp=sharing) _(for commenting)_
+* [Pull-Request for 0.8 API spec](https://github.com/GovStackWorkingGroup/BuildingBlockAPI/pull/15) _(Raise PR if you want to suggest a change)_
```

Comment:
Look to be linking to a PR?

Reply:
@benjaoming Can you please raise an issue for this when this is ready?